LICENSE

```diff
@@ -1,10 +1,290 @@
-NAME
-====
+GaiaX 4 PLC-AAD: Ontology Management
+====================================
 
-Copyright (C) YEAR & NAME
+All rights reserved. This program and the accompanying
+materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this
+distribution, and is available at
+http://www.eclipse.org/legal/epl-v20.html
 
+Contributors:
+- [<Initial Author> - <Description, e.g.: initial API and implementation>]
```

Comment:
Don't we want to add something like 
"PLC-AAD project partners" instead of <Initial Author>" and "update license" instead of "Description"?

Reply:
Mentioning yourself in the copyright notice is to my knowledge optional. Not all GaiaX project partners do actually contribute and everyone who does may feel free to leave its note here.

Reply:
@jdsika friendly reminder: what do you think?